relevanceai/http_client.py

```diff
@@ -100,3 +102,81 @@ def check_auth(self):
             return response.status_code == 200    
         except:
             raise Exception("Invalid auth details.")
+
+    def kmeans_cluster(self,
+        dataset_id: str,
+        vector_fields: list,
+        filters: List = [],
+        k: Union[None, int] = 10,
+        init: str = "k-means++",
+        n_init: int = 10,
+        max_iter: int = 300,
+        tol: float = 1e-4,
+        verbose: bool = True,
+        random_state = None,
+        copy_x: bool = True,
+        algorithm: str ="auto",
+        alias: str = "default",
+        cluster_field: str="_cluster_"
+    ):
+        """
+        This function performs all the steps required for Kmeans clustering:
+        1- Loads the data
+        2- Clusters the data
+        3- Updates the data with clustering info
+        4- Adds the centroid to the hidden centroid collection
+
+        input parameters:
```

Comment:
can you please make sure this is in same format as the others? Reason is because automated documentation follows a specific format and will need to adhere for consistency. 

https://github.com/RelevanceAI/RelevanceAI/blob/main/relevanceai/api/batch/batch_insert.py#L104